uniffi_bindgen/src/scaffolding/mod.rs

```diff
@@ -46,6 +48,8 @@ mod filters {
             Type::Optional(t) => format!("std::option::Option<{}>", type_rs(t)?),
             Type::Sequence(t) => format!("std::vec::Vec<{}>", type_rs(t)?),
             Type::Map(t) => format!("std::collections::HashMap<String, {}>", type_rs(t)?),
+            Type::External { .. } => panic!("External types coming to a uniffi near you soon!"),
+            Type::Wrapped { .. } => panic!("Wrapped types coming to a uniffi near you soon!"),
```

Comment:
Should we add `rust_type` to the enums?  I think we need to implement this in order to return the types directly from a function.

Reply:
Ooof, that's a good point - and one that I'm not quite sure how to resolve - we don't know the rust type for `External` types because we don't need to look in the "source" udl. Then I saw #1034 which is going to conflict with any changes here anyway, so I think I'm just going to land this with that limitation, just in the interests of allowing us to keep moving forward.